catalog/plugin/catalog-plugin-metacardingest-network/src/test/java/org/codice/ddf/catalog/plugin/metacard/util/AttributeHelperTest.java

```diff
@@ -0,0 +1,345 @@
+                .putAll(attributesTitleAndDescription)
+                .putAll(attributeMetadata)
+                .putAll(attributePoc)
+                .putAll(attributeCreated)
+                .build();
+        attributeHelper.applyNewAttributes(metacards, attributeMap);
+
+        assertThatMetacardHasExpectedTitleAndDescription(metacardWithPoc);
+        assertThatMetacardHasExpectedTitleAndDescription(metacardWithCreatedDate);
+
+        Map metadataResult = ImmutableMap.builder()
+                .putAll(attributesTitleAndDescription)
+                .putAll(attributePoc)
+                .putAll(attributeCreated)
+                .build();
+        assertThatMetacardHasAttributesEqualToThoseInMap(
+                metacardWithMetadataNoExpectedTitleOrDescription,
+                metadataResult);
+
+        Map pocResult = ImmutableMap.builder()
+                .putAll(attributeMetadata)
+                .putAll(attributeCreated)
+                .build();
+        assertThatMetacardHasAttributesEqualToThoseInMap(metacardWithPoc, pocResult);
+
+        Map createdResult = ImmutableMap.builder()
+                .putAll(attributeMetadata)
+                .putAll(attributePoc)
+                .build();
+        assertThatMetacardHasAttributesEqualToThoseInMap(metacardWithCreatedDate, createdResult);
+    }
+
+    @Test
+    public void testCreateAttributeInteger() throws Exception {
+        runParameterizedAttributeTest(INTEGER, "186", 186);
+    }
+
+    @Test
+    public void testCreateAttributeDouble() throws Exception {
+        runParameterizedAttributeTest(DOUBLE, "0.8866432", 0.8866432);
+    }
+
+    @Test
+    public void testCreateAttributeFloat() throws Exception {
+        runParameterizedAttributeTest(FLOAT, "0.123", 0.123f);
+    }
+
+    @Test
+    public void testCreateAttributeShort() throws Exception {
+        runParameterizedAttributeTest(SHORT, "16", (short) 16);
+    }
+
+    @Test
+    public void testCreateAttributeLong() throws Exception {
+        runParameterizedAttributeTest(LONG, "123456789", 123456789L);
+    }
+
+    @Test
+    public void testCreateAttributeBoolean() throws Exception {
+        runParameterizedAttributeTest(BOOLEAN, "true", true);
+    }
+
+    @Test
+    public void testCreateAttributeBinary() throws Exception {
+        String binaryString = "0100101010110110010010110110010";
+        runParameterizedAttributeTest(BINARY, binaryString, binaryString.getBytes());
+    }
+
+    @Test
+    public void testCreateAttributeDate() throws Exception {
+        String lexicalXsdDateTime = "2001-10-26T21:32:52";
+        runParameterizedAttributeTest(DATE,
+                lexicalXsdDateTime,
+                DatatypeConverter.parseDateTime(lexicalXsdDateTime)
+                        .getTime());
+    }
+
+    @Test
+    public void testCreateAttributeString() throws Exception {
+        runParameterizedAttributeTest(STRING, "This is a test String", "This is a test String");
```

Comment:
Couldn't the `SAMPLE_JSON` field be used in this test as well? As part of that, you could also rename `SAMPLE_JSON` to be `SAMPLE_JSON_STRING` to be more explicit/descriptive.